cratedb_toolkit/datasets/tutorial.py

```diff
@@ -36,8 +36,24 @@
     init_includes_loading=True,
 )
 
+windfarm_uk_info = Dataset(
+    reference="tutorial/windfarm-uk-info",
+    data_url="https://cdn.crate.io/downloads/datasets/cratedb-datasets/devrel/uk-offshore-wind-farm-data/wind_farms.json",
```

Comment:
Should we declare `https://cdn.crate.io/downloads/datasets/cratedb-datasets` as a constant and use it all over?

Reply:
Good idea. I will change it on one of the future iteration. Thanks.

Reply:
I've added e8b440f8 to GH-361.